package.json

```diff
@@ -26,6 +26,7 @@
     "lint": "npm run eslint && npm run stylelint",
     "prettier": "prettier --write '**'",
     "prettier-ci": "npm run prettier -- --list-different || (echo '\n\nThis failure means you did not run `yarn prettier` before committing\n\n' && exit 1)",
```

Comment:
Whoops, this line should give `yarn prettier-dev` as a hint for those who do not RTFM.

Reply:
Never knew what [RTFM](https://www.urbandictionary.com/define.php?term=RTFM) mean't until today 😅